lib/models/ObjectMDAmzRestore.js

```diff
@@ -0,0 +1,93 @@
+/*
+ * Code based on Yutaka Oishi (Fujifilm) contributions
+ * Date: 11 Sep 2020
+ */
+
+/**
+ * class representing the x-amz-restore of object metadata.
+ *
+ * @class
+ */
+class ObjectMDAmzRestore {
+    /**
+     *
+     * @constructor
+     * @param {boolean} ongoingRequest ongoing-request
+     * @param {Date} [expiryDate] expiry-date
+     * @throws {Error} case of invalid parameter
+     */
+    constructor(ongoingRequest, expiryDate = undefined) {
+        this.setOngoingRequest(ongoingRequest);
+        this.setExpiryDate(expiryDate);
+    }
+
+    /**
+     *
+     * @param {Object} data archiveInfo
+     * @returns {boolean} true if the provided object is valid
+     */
+    static isValid(data) {
+        try {
+            const object = new ObjectMDAmzRestore(data['ongoing-request'], data['expiry-date']);
+            return !!object;
```

Comment:
Why not return true since the constructor will throw if it's invalid?
```suggestion
            return true;
```

Reply:
I think it was mostly a linter issue with the current pattern: if `object` is never used we will have an error, but yes, you are right!

Reply:
I used the `instanceof` instead: https://github.com/scality/Arsenal/pull/1855/commits/2a4322a1e875def8892cf9575b8351af67a8db3a to make the code easier to read, what do you think?